insights/parsers/yum_updateinfo.py

```diff
@@ -0,0 +1,42 @@
+"""
+UpdateInfo - command ``yum updateinfo list -C``
+===============================================
+Provides a list of available advisories
+"""
+from insights import CommandParser, parser
+from insights.parsers import parse_delimited_table, SkipException
+from insights.specs import Specs
+
+
+@parser(Specs.yum_updateinfo)
+class YumUpdateinfo(CommandParser):
+    """
+    Class for parsing the output of `yum updateinfo list -C`.
+
+    Expected output of the command is::
+        RSHA-2020-0001 security       firefox-83.0-13.fc32.x86_64
```

Comment:
@semtexzv you need to add a blank line above line 17.  You can check the output to verify it is formatted correctly by following the [documentation guidelines](https://insights-core.readthedocs.io/en/latest/docs_guidelines.html#testing-your-examples).